src/components/measureGroups/MultipleSelectDropDown.tsx

```diff
@@ -1,11 +1,12 @@
-import React from "react";
+import React, { ReactNode } from "react";
 import tw from "twin.macro";
 import OutlinedInput from "@mui/material/OutlinedInput";
 import MenuItem from "@mui/material/MenuItem";
 import FormControl from "@mui/material/FormControl";
 import Select from "@mui/material/Select";
 import { Checkbox, Link, ListItemText } from "@mui/material";
 import { InputLabel } from "@madie/madie-design-system/dist/react/";
+import { boolean } from "yup/lib/locale";
```

Comment:
is this different from typescript boolean?

Reply:
It looks like some built in utility. vscode auto added it at some point by accident